src/app/tests/suites/certification/Test_TC_DISHALM_1_1.yaml

```diff
@@ -11,90 +11,109 @@
+              type: list
+              contains: [1]
 
-          The list MAY contain values in the Manufacturer Extensible Identifier (MEI) range: (0xXXXX_0000 - 0xXXXX_00FF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1), these values SHALL be ignored.
-
-          The list SHALL NOT contain any values in the Test Vendor or invalid range: (0x0000_0100 - 0x0000_FFFF), (0xXXXX_0100 - 0xXXXX_FFFF) and (0xFFF1_0000 - 0xFFFF_FFFF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1)
-      disabled: true
-
-    - label: "Step 6: TH reads from the DUT the AcceptedCommandList attribute."
-      verification: |
-          Verify that the DUT response contains the AcceptedCommandList attribute and have the list of Accepted Command:
-          The list MAY include these optional entries:
-          - 0x01: SHALL be included if and only if DISHALM.S.C01.Rsp(Modify)
-
-          The list include entries based on feature support:
-          - 0x00: SHALL be included if and only if DISHALM.S.C00.Rsp(Reset)
-          The list SHALL NOT contain any additional values in the standard or scoped range: (0x0000_0000 - 0x0000_00FF)
-
-          The list MAY contain values in the Manufacturer Extensible Identifier (MEI) range: (0xXXXX_0000 - 0xXXXX_00FF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1), these values SHALL be ignored.
-
-          The list SHALL NOT contain any values in the Test Vendor or invalid range: (0x0000_0100 - 0x0000_FFFF), (0xXXXX_0100 - 0xXXXX_FFFF) and (0xFFF1_0000 - 0xFFFF_FFFF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1)
-      disabled: true
-
-    - label: "Step 7: TH reads from the DUT the GeneratedCommandList attribute."
-      verification: |
-          Verify that the DUT response contains the GeneratedCommandList attribute and have the list of Generated Command:
-          This cluster has no entries in the standard or scoped range.
-
-          The list SHALL NOT contain any additional values in the standard or scoped range: (0x0000_0000 - 0x0000_00FF)
-
-          The list MAY contain values in the Manufacturer Extensible Identifier (MEI) range: (0xXXXX_0000 - 0xXXXX_00FF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1), these values SHALL be ignored.
-
-          The list SHALL NOT contain any values in the Test Vendor or invalid range: (0x0000_0100 - 0x0000_FFFF), (0xXXXX_0100 - 0xXXXX_FFFF) and (0xFFF1_0000 - 0xFFFF_FFFF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1)
-      disabled: true
+    - label:
+          "Read the attribute (Reset) based on feature support:
+          AcceptedCommandList"
+      PICS: DISHALM.S.C00.Rsp
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [0]
+
+    - label: "Read the global attribute: GeneratedCommandList"
+      command: "readAttribute"
+      attribute: "GeneratedCommandList"
+      response:
+          value: []
```

Comment:
This does not follow the test plan, which allows vendor-scoped things in here, right?  Followup, etc.

Reply:
Reference: https://github.com/project-chip/connectedhomeip/issues/28287